blox.cue

```diff
@@ -1,4 +1,5 @@
 {
-    data_dir: "tests/data"
-    schema_dir: "tests/schemas"
-}
\ No newline at end of file
+    data_dir: "dogfood/data/source"
```

Comment:
Figured it was better to use the dog food directory for tests than the contrived test cases